File: stressTest.py
# import the required libraries
import mysql.connector
import time

# define the MySQL connection settings
mydb = mysql.connector.connect(
  host="192.168.1.199",
  user="root",
  password="root",
  database="cad_imobiliarias"
)

# define the query to be executed
query = "SELECT * FROM cadastros"

# define the number of times to execute the query
num_iterations = 100

# execute the query for the given number of times
start_time = time.time() # start timer
for i in range(num_iterations):
    mycursor = mydb.cursor()
    mycursor.execute(query)
    myresult = mycursor.fetchall()
    for x in myresult:
        print(x)
end_time = time.time() # end timer

# calculate and print the time taken to execute the querie
print("Time taken to execute the query " + str(num_iterations) + " times: " + str(end_time - start_time) + " seconds")

File: stressTest.php
<?php
// define the MySQL connection settings
$servername = "localhost";
$username = "yourusername";
$password = "yourpassword";
$dbname = "yourdatabase";

// create connection
$conn = new mysqli($servername, $username, $password, $dbname);

// check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

// define the query to be executed
$query = "SELECT * FROM yourtable";

// define the number of times to execute the query
$num_iterations = 100;

// execute the query for the given number of times
$start_time = microtime(true); // start timer
for ($i = 0; $i < $num_iterations; $i++) {
    $result = $conn->query($query);
    if ($result->num_rows > 0) {
        while($row = $result->fetch_assoc()) {
            echo "id: " . $row["id"]. " - Name: " . $row["name"]. " - Email: " . $row["email"]. "<br>";
        }
    } else {
        echo "0 results";
    }
}
$end_time = microtime(true); // end timer

// calculate and print the time taken to execute the queries
echo "Time taken to execute the query " . $num_iterations . " times: " . ($end_time - $start_time) . " seconds";

// close connection
$conn->close();
?>
